yast2-iscsi-client.spec.in

```diff
@@ -7,6 +7,7 @@ BuildRequires:	docbook-xsl-stylesheets doxygen libxslt perl-XML-Writer popt-deve
 BuildRequires:  yast2 >= 2.17.93
 Requires:  yast2 >= 2.17.93
 Requires:	yast2-packager
+Requires:       open-iscsi
```

Comment:
Please add explanation `why` for the future. I know that now it's understandable if you read the bug.